app/enterprise/changelog.md

```diff
@@ -5,6 +5,127 @@ no_version: true
 skip_read_time: true
 ---
 
+## 2.1.3.0
+**Release Date** 2020/08/25
+
+Kong Enterprise 2.1.3.0 version includes 2.1.0.0 (beta) features, fixes, known issues, and workarounds. See the [2.1.0.0 (beta)](/enterprise/changelog/#2100-beta/) changelog for more details.
+
+### Features
+
+**Note: Feature updates for Kong Enterprise 2.1.3.0 version includes [2.1.0.0 (beta)](/enterprise/changelog/#features) features.**
+
+#### Kong Gateway
+* Inherited changes from OSS Kong in releases 2.0.x, 2.1.0, 2.1.1, 2.1.2, and 2.1.3.
+* Workspaces code has been refactored for performance. The feature should work the same for most users.
+* TLS version may be specified when using tls to connect to a Postgres db.
```

Comment:
`db` should be `DB` or `database`.  `tls` should be `TLS`

Reply:
Fixed.